.github/workflows/tpv2.yml

```diff
@@ -280,6 +280,22 @@ jobs:
         uses: actions/checkout@master
         with:
           fetch-depth: 5
+      - name: Cache node modules
+        id: restore-npm-cache
+        uses: actions/cache@v3
+        with:
+          path: ./experimental/traffic-portal/node_modules
+          key: ${{ runner.os }}-node-${{ hashFiles('./experimental/traffic-portal/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - name: Node 20
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.x
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 5
```

Comment:
ok, does it matter that one is checking out master and one is checking out v2?

Reply:
`@master` refers to https://github.com/actions/checkout/commits/master and `@v2` refers to https://github.com/actions/checkout/commits/v2.

It looks like the actions/checkout `master` branch has not been updated in 4 years, since they changed the default branch to `main`, so that should be updated to either `v2` or `main`.

Reply:
I chose main.